src/components/MovieCard/MovieCard.js

```diff
@@ -47,11 +47,11 @@ const MovieCard = ({ movie }) => {
         <></>
```

Comment:
Please follow the guides of [conditioanl rendering](https://reactjs.org/docs/conditional-rendering.html) and update the template:

```jsx
      {isDeleteMovieDialogVisible && (
        <DeleteMovieDialog
          setDeleteDialogVisibility={setIsDeleteMovieDialogVisible}
          movie={movie}
        />
      )}
```